src/main/java/com/commercetools/emailprocessor/model/TenantConfiguration.java

```diff
@@ -37,11 +38,13 @@ public TenantConfiguration() {
      * @param secret ctp project client secret
      * @param url    api endpoint url
      */
-    public TenantConfiguration(final String key, final String id, final String secret, final String url) {
+    public TenantConfiguration(final String key, final String id, final String secret, final String url,final String
```

Comment:
@ahalberkamp 
1. Why does this class implement `Cloneable`?
2. Do you need the Clone method?
3. Does this constructor need to be public? Seems like its not used except for unit tests, which are in the same package.
4. Space missing before last param
5. No annotation usage

Reply:
Why does this class implement Cloneable?
When a class implement the clone method, it should implemenent the clonable interfacd

Do we need the Clone method?
Yes, I`am using it inside the IT-Test, for duplicating configuration

Does this constructor need to be public? Seems like its not used except for unit tests, which are in the same package.
0c0a73a

Space missing before last param
0c0a73a

No annotation usage
0c0a73a

Reply:
It's a well-know fact that you should not do clone :) http://www.artima.com/intv/bloch13.html (the guy who wrote it is one of the Java creators) Additionally, you should not pollute your prod code only because of testing.

Reply:
cf08b2f5b0b797be1c593c5233a13272786ece9b